pkg/network/server.go

```diff
@@ -101,6 +102,7 @@ func (s *Server) Start(errChan chan error) {
 
 	go s.bQueue.run()
 	go s.transport.Accept()
+	setServerAndNodeVersions(s.UserAgent, strconv.Itoa(int(s.id)))
```

Comment:
You're not guaranteed to have 64-bit `int` to convert from `uint32` correctly, so it's better to use `FormatUint` here.

Reply:
done this way